cockatrice/src/settings/shortcuts_settings.h

```diff
@@ -411,6 +411,9 @@ class ShortcutsSettings : public QObject
         {"Player/aNextTurn", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Next Turn"),
                                          parseSequenceString("Ctrl+Return;Ctrl+Enter"),
                                          ShortcutGroup::Game_Phases)},
+        {"Player/aHide", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Hide Card (in Reveal Window)"),
```

Comment:
would this label also work without the parens?

Reply:
It would. I added the parens because it's not immediately obvious that `hide` is referring to hiding a revealed card.

Reply:
I mean removing the parens, keeping their contents, it's easier to understand to translators

Reply:
I can change it to `Hide Card that was Revealed`

Reply:
both of those sound great!

Reply:
the last one might be interpreted as the other reveal option, reveal top card of deck

Reply:
actually yeah in that case `Hide Card in Reveal Window` is probably the clearest